sdk-springboot/pom.xml

```diff
@@ -10,15 +10,27 @@
     <version>1.12.0-SNAPSHOT</version>
   </parent>
 
-  <artifactId>dapr-sdk-springboot</artifactId>
-  <packaging>jar</packaging>
+  <groupId>io.dapr.spring</groupId>
+  <artifactId>dapr-sdk-spring-parent</artifactId>
+  <packaging>pom</packaging>
   <version>1.12.0-SNAPSHOT</version>
```

Comment:
@artur-ciocanu we need to change this to 0.12.0-SNAPSHOT (we use the 0 to show that this artifact is not stable yet.

Reply:
Done